Dockerfile.oc-image

```diff
@@ -0,0 +1,4 @@
+#This dockerfile require an openshift installer binary in CWD
```

Comment:
not sure i understand why we need this image

Reply:
will remove it

Reply:
We use the same build Dockerfile for all our operations

Reply:
fixed